product_docs/docs/epas/15/reference/sql_reference/02_data_types/04_date_time_types.mdx

```diff
@@ -59,33 +64,33 @@ The first variation supported by EDB Postgres Advanced Server is `INTERVAL DAY T
 
  `p` specifies the precision of the `second` field.
 
-EDB Postgres Advanced Server interprets this value as as 1 day, 2 hours, 34 minutes, 5 seconds and 678 thousandths of a second:
+EDB Postgres Advanced Server interprets the following value as as 1 day, 2 hours, 34 minutes, 5 seconds and 678 thousandths of a second:
 
  `INTERVAL '1 2:34:5.678' DAY TO SECOND(3)`
 
-EDB Postgres Advanced Server interprets this value as 1 day and 23 hours:
+EDB Postgres Advanced Server interprets the following value as 1 day and 23 hours:
 
  `INTERVAL '1 23' DAY TO HOUR`
 
-EDB Postgres Advanced Server interprets this value as 2 hours and 34 minutes:
+EDB Postgres Advanced Server interprets the following value as 2 hours and 34 minutes:
 
  `INTERVAL '2:34' HOUR TO MINUTE`
 
-EDB Postgres Advanced Server interprets this value as 2 hours, 34 minutes, 56 seconds and 13 thousandths of a second. The fractional second is rounded up to 13 because of the specified precision.
+EDB Postgres Advanced Server interprets the following value as 2 hours, 34 minutes, 56 seconds and 13 thousandths of a second. The fractional second is rounded up to 13 because of the specified precision.
 
  `INTERVAL '2:34:56.129' HOUR TO SECOND(2)`
 
 The second variation supported by EDB Postgres Advanced Server that's compatible with Oracle databases is `INTERVAL YEAR TO MONTH`. This variation stores a time interval in years and months.
 
-EDB Postgres Advanced Server interprets this value as 12 years and 3 months:
+EDB Postgres Advanced Server interprets the following value as 12 years and 3 months:
 
  `INTERVAL '12-3' YEAR TO MONTH`
 
-EDB Postgres Advanced Server interprets this value as 12 years and 3 months:
+EDB Postgres Advanced Server interprets the following value as 12 years and 3 months:
 
  `INTERVAL '456' YEAR(2)`
 
-EDB Postgres Advanced Server interprets this value as 25 years:
+EDB Postgres Advanced Server interprets the following value as 25 years:
```

Comment:
@nidhibhammar, @josh-heyer, @djw-m - Following this section, PG has a section on special values they support. Is that something we support and should be describing?

PG also has large sections on Time Zones and Interval Input and Interval Output. Should we be providing comparable information? See [Date/Time types](https://www.postgresql.org/docs/current/datatype-datetime.html)

Reply:
We should, either by inclusion or by reference.

Reply:
I brought the sections on Time Zones, Interval Input, and Interval Output from PG and added them to our documentation after some minor editing.